test/es6/rlexe.xml

```diff
@@ -971,12 +969,6 @@
       <compile-flags>-es6spread</compile-flags>
     </default>
   </test>
-  <test>
-    <default>
-      <files>OS_917200.js</files>
-      <compile-flags>-ES6Destructuring-</compile-flags>
-    </default>
-  </test>
```

Comment:
This test specifically tested behaviour when destructuring is disabled - with the switch removed this behaviour no longer exists, hence test removed.